actualColorcodePair.js

```diff
@@ -0,0 +1,29 @@
+const ActualColorCodePairs = [
+    ' 1 | White  | Blue  ',
+  ' 2 | White  | Orange',
+  ' 3 | White  | Green ',
+  ' 4 | White  | Brown ',
+  '5  | White  | Slate ',

```

Comment:
There is a misalignment between 4, 5, 10,11.
The reference alignment values are used to compare against output, this needs to be corrected.
The test case seems (I could have misread and wrong) to be written in such a way that the misalignment is allowed ! 
Please check and correct.